src/patternfly/utilities/Text/text.scss

```diff
@@ -0,0 +1,47 @@
+// stylelint-disable
+
+// Font size options
+$pf-u-font-size-options: (
+  font-size-xs:  (font-size var(--pf-global--FontSize--xs)),
+  font-size-sm:  (font-size var(--pf-global--FontSize--sm)),
+  font-size-md:  (font-size var(--pf-global--FontSize--md)),
+  font-size-lg:  (font-size var(--pf-global--FontSize--lg)),
+  font-size-xl:  (font-size var(--pf-global--FontSize--xl)),
+  font-size-2xl: (font-size var(--pf-global--FontSize--2xl)),
+  font-size-3xl: (font-size var(--pf-global--FontSize--3xl)),
+  font-size-4xl: (font-size var(--pf-global--FontSize--4xl))
+);
+
+@include pf-utility-builder($pf-u-font-size-options, $pf-global--breakpoint-list);
+
+
+// Font weight options
+$pf-u-font-weight-options: (
+  font-weight-light:              (font-weight var(--pf-global--FontWeight--light)),
+  font-weight-normal:             (font-weight var(--pf-global--FontWeight--normal)),
+  font-weight-bold:               (font-weight var(--pf-global--FontWeight--bold)),
+);
+
+@include pf-utility-builder($pf-u-font-weight-options, $pf-global--breakpoint-list);
+
+
+// Font color options
+$pf-u-font-color-options: (
+  color-100: (color var(--pf-global--Color--100)),
+  color-200: (color var(--pf-global--Color--200)),
+  color-300: (color var(--pf-global--Color--300)),
+  color-400: (color var(--pf-global--Color--400)),
```

Comment:
I think we'll also want to add

* pf-global--icon--Color--light
* pf-global--icon--Color--dark
* pf-global--link--Color
* pf-global--active-color--[100-400]
* pf-global--disabled-color--[100-300]
* pf-global--primary-color--100
* pf-global--primary-color--200
* pf-global--secondary-color--100
* pf-global--default-color--100
* pf-global--default-color--200
* pf-global--default-color--300
* pf-global--success-color--100
* pf-global--success-color--200
* pf-global--info-color--100
* pf-global--info-color--200
* pf-global--warning-color--100
* pf-global--warning-color--200
* pf-global--danger-color--100
* pf-global--danger-color--200
* pf-global--danger-color--300

We may consider adding the palette vars and light/dark theme vars, but that can probably wait until someone requests them.